AuxTel/AuxTel-Standard-Operations/AuxTel-Daytime-Operations/latiss-daily-calibrations-full.rst

```diff
@@ -0,0 +1,277 @@
+
+.. code-block:: text
+  :caption: take_image_latiss.py
+
+    image_type: FLAT
+    filter: SDSSr_65mm
+    grating: empty_1
+    reason: daily_PTC
+    exp_times:
+            0.25,
+            0.25,
+            1.42,
+            1.42,
+            6.53,
+            6.53,
+            4.23,
+            4.23,
+            30.04,
+            30.04,
+            12.56,
+            12.56,
+            57.75,
+            57.75,
+            8.13,
+            8.13,
+            2.73,
+            2.73,
+            3.40,
+            3.40,
+            1.77,
+            1.77,
+            111.03,
+            111.03,
+            37.35,
+            37.35,
+            0.48,
+            0.48,
+            0.59,
+            0.59,
+            10.10,
+            10.10,
+            1.14,
+            1.14,
+            0.20,
+            0.20,
+            89.29,
+            89.29,
+            71.81,
+            71.81,
+            0.38,
+            0.38,
+            0.31,
+            0.31,
+            19.43,
+            19.43,
+            2.20,
+            2.20,
+            15.62,
+            15.62,
+            0.92,
+            0.92,
+            0.74,
+            0.74,
+            24.16,
+            24.16,
+            5.25,
+            5.25,
+            46.44,
+            46.44
+
+
+**Set 7: Sequence for daily_flats.**
+
+.. code-block:: text
+  :caption: take_image_latiss.py
+
+    image_type: FLAT
+    filter: SDSSr_65mm
+    grating: empty_1
+    reason: daily_sflat,
+    exp_times:
+                    0.5,
+                    0.5,
+                    0.5,
+                    0.5,
+                    0.5,
+                    12.8,
+                    12.8,
+                    12.8,
+                    12.8,
+                    12.8
+
+
+The `latiss_daily_calibrations.py`  BLOCK finishes with the `power_off_atcalsys.py` SAL script with no configuration. It will turn off the lamp, close the shutter and shutdown the chiller. At this stage, the script completion time is 15 minutes.
+
+.. code-block:: text
+  :caption: power_off_atcalsys.py
+
+Finally, the `set_summary_state.py` script sends ATWhiteLight back to ``STANDBY``.
+
+.. code-block:: text
+  :caption: set_summary_state.py
+
+    data:
+   -
+     - ATWhiteLight 
+     - STANDBY
+
+Once this last script is done, check the camera to make sure the white light is off. At this point, ATCS and LATISS are in ``ENABLED`` state and the dome and telescope are in FLAT position.
+
+.. note::
+   The location of the `BLOCK source code`_ can be checked and if the filters or exposures times have changed, create a ticket for yourself and edit this document accordingly.
+
+
+Contingency
+===========
+In cases such as: telescope and dome movement not allowed or not cleared, not enough time available for calibrations; proceed to only take BIAS and DARKS following the procedure in LATISS Daily Calibrations (BIAS and DARKS only).
```

Comment:
I think we should recommend skipping calibrations all together in these cases (including BIAS and DARKS).